tooling/minitest/src/tests/wide_ptr.rs

```diff
@@ -0,0 +1,61 @@
+use crate::*;
+use tests::slice::ref_as_slice;
+
+#[test]
+fn get_metadata_slice() {
```

Comment:
Please move this into the `slice` file, rather than importing the helper function from there. (That helper should be private.)